setup/acr/README.md

```diff
@@ -1,63 +0,0 @@
-# Connaisseur with AKS and ACR
```

Comment:
Do we want to scrap this without replacement?

Reply:
no, in general things were moved over. So for most contents I copied them over into other parts of the docu. The essence of this guide should be briefly summarized in https://github.com/sse-secure-systems/connaisseur/blob/docs/documentation-2.0/docs/validators/notaryv1.md#using-azure-container-registry
I would probably recommend to drop the full setup instructions